Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM golang:1.13-alpine3.11 AS builder
 
+RUN apk add -U --no-cache \
+        ca-certificates
+
+WORKDIR /go/src/github.com/minchao/go-realworld
+COPY go.mod go.sum ./
+RUN go mod download
+
+# build application
 ARG BUILD_VERSION
 ARG BUILD_DATE
 ARG BUILD_COMMIT
 ARG CMD_PACKAGE
 
-WORKDIR /go/src/github.com/minchao/go-realworld
 COPY . .
-RUN apk add -U --no-cache \
-        ca-certificates
```

Comment:
Always get latest version of ca-certificates.
Maybe can add version to package like`ca-certificates=20191127-r0`?

Reply:
update with base image